draft-tigress-requirements.md

```diff
@@ -65,128 +70,123 @@ This document describes the use cases necessitating the secure transfer of digit
 - (Req-RoundTrips) Solution shall allow for multiple data exchanges between sender and receiver devices in the process of credential transfer. This requirement shall align with (Req-Connectivity) above.
-- (Req-Speed) When both Sender and Receiver are online at the same time they should be able to quickly and efficiently transfer data.
 - (Req-Opaque) In the case when an intermediary server is used to facilitate the credential transfer, message content between sender and receiver must be opaque to an intermediary, intermediary server shall not be able to recognize the content of provisioning information or use it to provision digital credential on its own.
-- (Req-Invitation) The Receiver must be able to establish a connection with the Sender for the secure credential transfer using an invite that can be sent over any generic communication channel (e.g. sms, email, NFC).
+
 
 # Security Considerations
 
-- Threat model for implementation that uses an intermediary server to facilitate the credential transfer should consider trusted relationship between a sender device and the intermediary server. Intermediary server shall be able to verify that the sender device is in good standing and content generated by the sender device can be trusted by the intermediary. The trust mechanism could be proprietary or publicly verifiable ( e.g. WebAuthN). This is important because intermediary server shall have no visibility to the content of the provisiong information sent through it (Req-Opaque).
+- Threat model for implementation that uses an intermediary server to facilitate the credential transfer should consider trusted relationship between a sender device and the intermediary server. Intermediary server shall be able to verify that the sender device is in good standing and content generated by the sender device can be trusted by the intermediary. The trust mechanism could be proprietary or publicly verifiable ( e.g. WebAuthN). This is important because intermediary server shall have no visibility to the content of the provisioning information sent through it (Req-Opaque).
 - Threat model for implementation that uses an intermediary server to facilitate the credential transfer, should consider evaluation of the trustworthiness of the intermediary server by the receiver device based on agreed criteria.
+- Tigress implementation shall avoid collecting user or device identities, as well as storing and using such identities for purpose other then the credential transfer itself.
+
+# General considerations
+
+- A single token of Provisioning Information shall be used for a single transfer of digital credential, shall not be redeemable for multiple additional transfer attempts. Sharing in a chain (when Receiver, after provisioning the credential to the device, becomes a new Sender using the same token of Provisioning Information) or sharing to multiple devices shall not be used with a single piece of Provisioning Information.
```

Comment:
This may be more of an implementation detail. Since the provisioning partner is out of scope, I'm not sure it makes sense to add this point in the requirements document. @alexpell00 @dimmyvi thoughts?

Reply:
This is general considerations for potential implementation - added here as it was discussed in interim IEFT meeting